### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by Nadine2016**
# Description

Please include a high level summary of the changes.

This bug fixes ... \ This new feature can be used to ...

_Fill out any of the following sections that are relevant and remove the others_
## API changes
- Added x field to y resource
- ...

## Code changes
- Fix error in `Foo()` function
- Add `Bar()` function
- ...

## CI changes
- Adjusted schedule for x job
- ...

## Docs changes
- Added guide about feature x to public docs
- Updated README to account for y behavior
- ...

# Context

Users ran into this bug doing ... \ Users needed this feature to ...

See slack conversation [here](https://solo-io-corp.slack.com/archives/some/post)

## Interesting decisions
 
We chose to do things this way because ...

## Testing steps

I manually verified behavior by ...

## Notes for reviewers

Be sure to verify intended behavior by ...

Please proofread comments on ...

This is a complex PR and may require a huddle to discuss ...

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/content/guides/security/cors/_index.md [hunk 1]
```diff
@@ -4,23 +4,25 @@ weight: 70
...
+
+Note that the preflight request is optional. Web browsers can also be configured to send the cross-origin directly. However, access to the request resource is granted only if CORS headers were returned in the response. If no headers are returned during the preflight request, the web browser denies access to the resource in the other domain. 
+
+CORS policies are typically implemented to limit access to server resources for JavaScripts that are embedded in a web page, such as
```
Comment (Rachael-Graham) [inline docs/content/guides/security/cors/_index.md:17]:
```suggestion
CORS policies are typically implemented to limit access to server resources for JavaScripts that are embedded in a web page, such as:
```

File: docs/content/guides/security/cors/_index.md [hunk 2]
```diff
@@ -4,23 +4,25 @@ weight: 70
...
+
+Note that the preflight request is optional. Web browsers can also be configured to send the cross-origin directly. However, access to the request resource is granted only if CORS headers were returned in the response. If no headers are returned during the preflight request, the web browser denies access to the resource in the other domain. 
+
+CORS policies are typically implemented to limit access to server resources for JavaScripts that are embedded in a web page, such as 
+* A JavaScript on a web page at `example.com` tries to access a different domain, such as `api.com`.
+* A JavaScript on a web page at `example.com` tries to access a different subdomain, such as `api.exampl.com`.
```
Comment (Rachael-Graham) [inline docs/content/guides/security/cors/_index.md:19]: just making sure that `exampl` is intentionally misspelled? wasnt sure if that was the different domain, or if the `api.` is the diff part

File: docs/content/guides/security/cors/_index.md [hunk 3]
```diff
@@ -4,23 +4,25 @@ weight: 70
...
+* A JavaScript on a web page at `example.com` tries to access a different port, such as `example.com:3001`.
+* A JavaScript on a web page at `https://example.com` tries to access the resources by using a different protocol, such as `http://example.com`. 
 
 For more details, see [this article](https://medium.com/@baphemot/understanding-cors-18ad6b478e2b).
 
-### Where to Use It
+## How to configure a VirtualService for CORS?
```
Comment (Rachael-Graham) [inline docs/content/guides/security/cors/_index.md:25]:
```suggestion
## How to configure a VirtualService for CORS
```

File: docs/content/guides/security/cors/_index.md [hunk 4]
```diff
@@ -41,7 +43,9 @@ spec:
     - '*'
 {{< /highlight >}}
 
-{{< notice note >}} Some apps, such as `httpbin`, have built-in CORS policies that allow all origins. These policies take precedence over CORS policies that you might configured in Gloo Edge.  {{< /notice >}}
+{{% notice note %}} 
+Some apps, such as `httpbin`, have built-in CORS policies that allow all origins. These policies take precedence over CORS policies that you might configured in Gloo Edge.
```
Comment (Rachael-Graham) [inline docs/content/guides/security/cors/_index.md:47]:
```suggestion
Some apps, such as `httpbin`, have built-in CORS policies that allow all origins. These policies take precedence over CORS policies that you might configure in Gloo Edge. 
```

File: docs/content/guides/security/cors/_index.md [hunk 5]
```diff
@@ -50,54 +54,126 @@ The following fields are available when specifying CORS on your `VirtualService`
...
 | `allowMethods`     | `[]string` | Specifies the content for the *access-control-allow-methods* header.                                                                                             |         |
 | `allowHeaders`     | `[]string` | Specifies the content for the *access-control-allow-headers* header.                                                                                             |         |
 | `exposeHeaders`    | `[]string` | Specifies the content for the *access-control-expose-headers* header.                                                                                            |         |
 | `maxAge`           | `string`   | Specifies the content for the *access-control-max-age* header.                                                                                                   |         |
 | `allowCredentials` | `bool`     | Specifies whether the resource allows credentials.                                                                                                               |         |
+| `disableForRoute` | `bool` | If set, the CORS Policy (specified on the virtual host) will be disabled for this route. | false |
```
Comment (Rachael-Graham) [inline docs/content/guides/security/cors/_index.md:63]:
```suggestion
| `disableForRoute` | `bool` | If set, the CORS Policy (specified on the virtual host) is disabled for this route. | false |
```

**Review by Rachael-Graham — APPROVED** [review]

**Review by artberger — APPROVED** [review]

**Pull request merged**